embedchain/embedchain.py

```diff
@@ -243,7 +243,7 @@ def dry_run(self, input_query, config: QueryConfig = None):
         if config is None:
             config = QueryConfig()
         context = self.retrieve_from_database(input_query)
-        prompt = self.generate_prompt(input_query, context)
+        prompt = self.generate_prompt(input_query, context, config.template)
```

Comment:
Hey @ianupamsingh : didnt understand this code? can you please explain.

Reply:
He's fixing a bug that found its way into prod: Dry run does not use the template config. This is ok.

Reply:
although in the future this should be a separate PR.

Reply:
If I'm quick I'll open a separate PR for that. This needs to be fixed asap.

Reply:
addressed in #203, I hope you don't mind. but this needs to be discussed separately.

Reply:
this is resolved.